src/main/java/com/robertx22/age_of_exile/event_hooks/ontick/OnClientTick.java

```diff
@@ -30,6 +36,19 @@ public static void onEndTick(Minecraft mc) {
         try {
             Player player = Minecraft.getInstance().player;
 
+            container.values().forEach(x -> {
```

Comment:
Why would you call talent tree code if the player doesn't even have the tree open?

Reply:
both tasks can't run if the player hasn't opened the tree screen yeah.